rep-2000.rst

```diff
@@ -418,6 +418,112 @@ Build System Support:
 - setuptools
 
 
+Eloquent Elusor (November 2019 - November 2020)
+---------------------------------------------------------------
+
+Targeted platforms:
+
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+| Architecture | Ubuntu Bionic (18.04) |     MacOS Mojave (10.14)     |   Windows 10 (VS2019)  | Debian Stretch (9)     |
```

Comment:
The latest Debian distro is Buster.

Reply:
Fixed, thanks.

Reply:
The information in the "Dependency Requirements" table needs to be updated accordingly.

Reply:
Updated, easiest to look at this diff: https://github.com/ros-infrastructure/rep/pull/203/commits/55ea92ed9213dc52afe65c5978794a6da5b5e2e7